src/actions/userActions.js

```diff
@@ -40,6 +40,7 @@ export function logoutError(errorMessage) {
 // refreshUserInfo performs the /v4/user/ call and updates what Happa knows
 // about the user based on the response.
 export function refreshUserInfo() {
```

Comment:
it's the localstorage call in here that fails and eventually causes `Cannot read property 'origin' of undefined` to bubble up.